Dijkstra Algorithm/Dijkstra.playground/Contents.swift

```diff
@@ -0,0 +1,137 @@
+//: Playground - noun: a place where people can play
+
+import Foundation
+import UIKit
+
+open class Vertex: Hashable, Equatable {
+
+    open var identifier: String!
+    open var neighbors: [(Vertex, Double)] = []
+    open var pathLengthFromStart: Double = Double.infinity
```

Comment:
The `Double` type can be inferred from `Double.infinity`, no need to do explicit typing.